tests/ui/test_ui_tools.py

```diff
@@ -966,7 +966,8 @@ def test_users_view(self, users, users_btn_len, editor_mode, status,
                 view=self.view,
                 width=width,
                 color=self.view.users[0]['status'],
-                count=1
+                count=1,
+                is_current_user=False
```

Comment:
I think it would be worth adding a `FIXME` comment here; this could be where we put the test later, or maybe nearby.